CONTRIBUTING.md

```diff
@@ -152,8 +152,11 @@ HTML Sphinx generated should be in `docs/_build/html`. Navigate there
 and start a simple web server so we can check out the docs in a browser:
 
 ```bash
-cd docs/_build/html
+cd _build/html
+#python2
```

Comment:
We don't support py2.7 anymore, so might as well just use `python -m http.server 8000`. You may want to add a note about py3, or straight up add the 3 in the command as in `python3 -m http.server 8000`. But I'm not sure whether `python3` symlink is available in all/most envs...

Reply:
>  But I'm not sure whether python3 symlink is available in all/most envs...

emmmm , i don't know how to guess symlink if available ,maybe virtualenv is a solution.

Reply:
> We don't support py2.7 anymore, so might as well just use python -m http.server 8000. You may want to add a note about py3, or straight up add the 3 in the command as in python3 -m http.server 8000.
I change the comment  as you wish.